definitions/specs/elasticsearch.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "Elasticsearch specs",
+  "description": "Common metadata for Elasticsearch clusters.",
+  "examples": [
+    {
+      "version": "7.17.1"
+    }
+  ],
+  "required": [
+    "version"
+  ],
+  "properties": {
+    "version": {
+      "type": "string",
+      "title": "Version",
+      "description": "The Elasticsearch version.",
+      "default": "",
```

Comment:
```suggestion
```

This is effectively a no-op. The default is always `""`.

Not to self: We should create a `semver.json` type. for these sorts of fields.